BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -388,7 +411,7 @@ extension AttributedLabel {
             let textBoundingBox = layoutManager.usedRect(for: textContainer)
             let textContainerOffset = CGPoint(
                 x: (labelSize.width - textBoundingBox.size.width) * alignmentMultiplier - textBoundingBox.origin.x,
-                y: (labelSize.height - textBoundingBox.size.height) * alignmentMultiplier - textBoundingBox.origin.y
+                y: (labelSize.height - textBoundingBox.size.height) * 0.5 - textBoundingBox.origin.y
```

Comment:
I added the tap detection snapshot tests, and it found that this has always been broken for any visually stretched label that wasn't _also_ centered. Oops!